src/debugAdapter/goDebug.ts

```diff
@@ -1371,6 +1381,39 @@ class GoDebugSession extends LoggingDebugSession {
 		return returnValue;
 	}
 
+	private async evaluateRequestFetch(args: DebugProtocol.EvaluateArguments, scope: any, delveLoadConfig: any): Promise<LoadConfig> {
+		if (!(!this.delve.isApiV1 && (args.context === 'variables'))) {
```

Comment:
Let's simplify this by `if (this.delve.isApiV1 || args.context !== 'variables')`